test/unit-test/core_sntp_client_utest.c

```diff
@@ -727,10 +740,17 @@ void test_Sntp_SendTimeRequest_ErrorCases()
                                                                       * take for holding auth data. */
     TEST_ASSERT_EQUAL( SntpErrorAuthFailure,
                        Sntp_SendTimeRequest( &context, rand() % UINT32_MAX ) );
+}
 
-    /* Reset authentication code size variable. */
-    authCodeSize = sizeof( testBuffer ) - SNTP_PACKET_BASE_SIZE;
-    expectedBytesToSend = SNTP_PACKET_BASE_SIZE + authCodeSize;
+/**
+ * @brief Validate the behavior of @ref Sntp_SendTimeRequest when failures occur in
+ * transport send operation.
+ */
+void test_Sntp_SendTimeRequest_Transport_Send_Failures()
```

Comment:
This test case should be broken into multiple tests.

Reply:
Okay, have broken the tests further.